src/lib/updateGrantsRecipients.test.js

```diff
@@ -2,6 +2,7 @@
 import { Op, QueryTypes } from 'sequelize';
 import axios from 'axios';
 import fs from 'mz/fs';
+import { expect } from '@playwright/test';
```

Comment:
you can remove this (we don't want the expect from playwright, we want to use the jest one)

Reply:
done